### PR TITLE
[22.05] Fixes for JupyTool, and overlapping versions with jupyter.

### PR DESCRIPTION
**Pull request by blankenberg**
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Review by hexylena — APPROVED** [review]

File: lib/galaxy/config/sample/tool_conf.xml.sample [hunk 1]
```diff
@@ -136,7 +136,7 @@
     <tool file="interactive/interactivetool_cellxgene.xml" />
     <tool file="interactive/interactivetool_ethercalc.xml" />
     <tool file="interactive/interactivetool_hicbrowser.xml" />
-    <tool file="interactive/interactivetool_jupyter_notebook_0.3.xml" />
+    <tool file="interactive/interactivetool_jupyter_notebook_1.x.x.xml" />
```
Comment (mvdbeek) [inline lib/galaxy/config/sample/tool_conf.xml.sample]: Can you put this at a fixed version please ?

Comment (blankenberg) [reply lib/galaxy/config/sample/tool_conf.xml.sample]: It is the 1.x.x series.

Comment (mvdbeek) [reply lib/galaxy/config/sample/tool_conf.xml.sample]: That's fine, but please call it `interactive/interactivetool_jupyter_notebook_1.0.0.xml`, we'll have to ship older versions as well.

Comment (blankenberg) [reply lib/galaxy/config/sample/tool_conf.xml.sample]: I think its a great idea to ship older version, but we haven't been doing that at all for any interactive tools.

**Pull request merged**